paper-input-behavior.html

```diff
@@ -349,6 +360,46 @@
     },
 
     /**
+     * Forward focus to inputElement
+     */
+    _onFocus: function() {
+      if (!this._shiftTabPressed) {
+        // iron-autogrow-textarea
+        if (this.inputElement.textarea) {
+          this.inputElement.textarea.focus();
+        } else {
+          this.inputElement.focus();
+        }
+      }
+    },
+
+    /**
+     * Forward blur to inputElement
+     */
+    _onBlur: function() {
```

Comment:
Does this work in this case of calling the `blur` method on the `paper-input`? There doesn't seem to be a test for that.